docs/apache_spark.md

```diff
@@ -4,40 +4,30 @@ template: full_width_with_menu.html
 
 # Apache Spark
 
-## SWAN and Spark Service at CERN
+## How to use Apache Spark with the CERN SWAN service
+This gallery shows examples of use of [Apache Spark][spark] within SWAN.
 
-These notebooks exemplify the usage of SWAN and Spark for analytics and machine learning use cases at CERN.
+[<img class="open_in_swan" data-path="apache_spark1" data-name="SWAN: Spark connector and monitor" alt="Open this Gallery in SWAN" src="https://swanserver.web.cern.ch/swanserver/images/badge_swan_white_150.png">][gallery_url]
 
-[<img class="open_in_swan" data-path="apache_spark1" data-name="SWAN: Spark connector and monitor" alt="Open this Gallery in SWAN" src="https://swanserver.web.cern.ch/swanserver/images/badge_swan_white_150.png">][gallery_url1]
+## These notebooks show how to use Spark for data processing and machine learning
```

Comment:
Do we want these titles to start with "These notebooks XXX" ?

Reply:
Good point, I'll restructure the title.